lib/vaahextendflutter/widgets/atoms/buttons.dart

```diff
@@ -341,10 +341,7 @@ class ButtonTextWithIcon extends StatelessWidget {
           fontSize: fontSize,
         ),
       ),
-      icon: FaIcon(
-        iconData,
-        size: iconSize,
-      ),
+      icon: leading!,
```

Comment:
this is buggy, what if someone passes null leading

Reply:
Made leading as required, because these widgets are called ButtonElevatedWithIcon and ButtonTextWithIcon. So, that makes leading icon/asset necessary.

Reply:
Yep, now that's correct. cheers!